Assets/MixedRealityToolkit.Examples/Experimental/Dialog/DialogExampleController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
```

Comment:
Please add license header (// Licensed under the MIT License.). If desired, feel free to add a copyright designation.